examples/README.md

```diff
@@ -0,0 +1,60 @@
+# examples
+
+This directory contains example code for both installing Contour and its associated Envoy Pods under various conditions, and also for setting up various extensions and other pieces of software that may help with running Contour.
+
+## Directories
+
+Most of the directories contain a complete set of Kubernetes YAML that can be applied to a cluster. This section describes the purpose of each directory, and inside each directory is a README.md that describes the details.
+
+### [`contour`](./contour/README.md)
+
+This is the recommended example installation of Contour, and will deploy Contour into a Deployment, and Envoy into a Daemonset, with the gRPC communication secured with certificates, and a `LoadBalancer` Service to expose Envoy to your upstream load balancer provider.
+
+### [`contour-dev`](./contour-dev/README.md)
+
+This is an example installation of Contour, designed for local development or other trial use. It will deploy a single Pod that runs a Contour and an Envoy container, with no secured gRPC between them (since they are using `localhost` to talk to each other.)
+
+### `common`
+
+This contains the YAML fragments that are common across multiple examples. Not for applying to a Kubernetes cluster directly.
+
+### `example-workload`
+
+This contains an example workload using Ingress ([kuard.yaml](./example-workload/kuard.yaml)) and one using IngressRoute ([kuard-ingressroute.yaml](./example-workload/kuard-ingressroute.yaml)).
+
+There are also further IngressRoute examples under the `example-workload/ingressroute` directory. See the [README](./example-workload/ingressroute/README.md) for more details on each example.
+
+### `grafana`, `prometheus`
+
+Contain Grafana and Prometheus examples, including the apps themselves, which can show the metrics that Contour exposes.
+
+If you have your own Grafana and Prometheus deployment already, the supplied [ConfigMap](./grafana/02-grafana-configmap.yaml) contains a sample dashboard with Contour's metrics.
+
+### `kind`, `root-rbac`
+
+Both of these examples are fragments used in other documentation ([deploy-options](../docs/deploy-options.md) and [ingressroute](../docs/ingressroute.md) respectively.)
+
+### `deployment-grpc-v2`
+
+> This example is deprecated and will be removed as part of the Contour 1.0 release.
+
+Installs Contour and Envoy in a single pod, as a Deployment.
```

Comment:
If we're deprecating it lets not tell people what they're missing. Never look back darling, it distracts from the now.